lib/widgets/registerorgform.dart

```diff
@@ -113,12 +117,13 @@ class _RegisterOrgFormState extends State<RegisterOrgForm> {
                     },
                   ),
                 ]);
-          });
-      final user = await CurrentUser().get();
-      await Gateway().addAdmin(user.userId);
-      await CurrentUser().update();
-      return "";
+          }
+      );
+      return null;
```

Comment:
Is there another option than returning null? Does the call to registerAdmin() handle nulls and empty strings?